web/wp-content/mu-plugins/custom/lfevents/admin/partials/sync-sched.php

```diff
@@ -82,6 +82,7 @@
 			$speaker_query->the_post();
 			$speaker_id = get_the_ID();
 			update_post_meta( $speaker_id, $sched_event_id, json_encode( $speaker ) );
+			update_option( 'lfevents_sync_sched_last_run', current_time( 'timestamp' ) );
```

Comment:
Do we want this inside both loops like this? Once we have several events hooked up, it will run possibly 1000s of times each time we sync sched data?

Reply:
Ah. We should move it out the loop for sure. Just when the process finishes once.

Reply:
I'll fix